qcodes/instrument_drivers/QuTech/F1d.py

```diff
@@ -49,17 +56,21 @@ def __init__(self, name, spi_rack, module, **kwargs):
 
         self.add_parameter('RF_level',
                            label='RF level',
-                           set_cmd=self.f1d.get_RF_level,
-                           units='dBm')
+                           get_cmd=self.f1d.get_RF_level,
+                           unit='dBm')
 
         self.add_parameter('LO_level',
                            label='LO level',
-                           set_cmd=self.f1d.get_LO_level,
-                           units='dBm')
-
-        self.add_function('enable_remote', call_cmd=self.f1d.enable_remote)
-        self.add_function('clear_rf_clip', call_cmd=self.f1d.clear_rf_clip)
-        self.add_function('is_rf_clipped', call_cmd=self.f1d.rf_clipped)
+                           get_cmd=self.f1d.get_LO_level,
+                           unit='dBm')
+
+        self.add_parameter('enable_remote',
```

Comment:
why does this parameter not have a "get" command?

Reply:
@astafan8 This is due to hardware limitations

Reply:
how do you then know if the "remote" is enabled or not?